src/com/amplitude/api/Utils.java

```diff
@@ -103,4 +106,20 @@ static String normalizeInstanceName(String instance) {
         }
         return instance.toLowerCase();
     }
+
+    static SharedPreferences getAmplitudeSharedPreferences(Context context, String instanceName) {
+        assert !isEmptyString(instanceName);
+        String prefName = Constants.PACKAGE_NAME + "." + instanceName + "." + context.getPackageName();
```

Comment:
I assume this is similar to how we scope the DB data?

Reply:
Ya this is how we name our db file: https://github.com/amplitude/Amplitude-Android/blob/master/src/com/amplitude/api/DatabaseHelper.java#L74

This is how we named our legacy shared prefs file: https://github.com/amplitude/Amplitude-Android/blob/master/src/com/amplitude/api/AmplitudeClient.java#L2201-L2219